File: mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //asking for height
    int height;
    do
    {
        printf("What should the height of the pyramid be \n ");
        height = get_int();
    }
    while (height < 0 || height > 23);


    //building pyramid
    for (int row = 0; row < height; row++)
    {
        for (int spaces = height - row; spaces > 1; spaces--)
        {
            printf(" ");
        }
        for (int hashtags = 0; hashtags < row + 1; hashtags++)
        {
            printf("#");
        }

        printf("  ");

        for (int hashtags = 0; hashtags < row + 1; hashtags++)
        {
            printf("#");
        }
        printf("\n");
    }









}

File: sentimental-mario-more/mario.py
from cs50 import get_int


while True:
    h = get_int("Height of marios pyramid: ")
    if h >= 0 and h <= 23:
        break

row = 0
hashtags = 0
spaces = h - row
for row in range(h):
    # spaces
    for spaces in range((h - row) - 1):
        print(" ", end="")
    # hashtags
    for hashtags in range(row+1):
        print("#", end="")

    # space between pyramids
    print("  ", end="")

    # second pyramid
    for hashtags in range(row+1):
        print("#", end="")
    print("")
